recipe/meta.yaml

```
@@ -230,336 +231,338 @@ outputs:
-      ignore_run_exports:
-        - arrow-cpp {{ arrow_version }}
-        - blosc
-        - cudatoolkit    # [cuda_compiler_version not in (undefined, "None")]
-        - double-conversion
-        - fmt
-        - glog
-        - libarchive
-        - libgdal
-        - libllvm{{ llvm_version }}
-        - libpng
-        - librdkafka
-        - libthrift {{ thrift_version }}
-        - pyarrow {{ arrow_version }}
-        - python
-        - tbb
-      run_exports:
-        - {{ pin_subpackage('pyomniscidbe',  max_pin='x.x.x') }}
-    requirements:
-      build:
-        # c compiler is specified here to get run constraint pins correct, presumably..
-        - {{ compiler('c') }}
-        - {{ compiler('cgo') }}
-        - {{ compiler('cxx') }}
-        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
-        - clangdev {{ llvm_version }}
-        # clang++ is used for generating the bytecodes of extension functions
-        - clangxx {{ llvm_version }}
-        - cmake
-        - llvmdev {{ llvm_version }}
-        - make
-        - ninja
-        - maven
-      host:
-        - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
-        - bisonpp
-        - blosc
-        - boost-cpp
-        - clangdev {{ llvm_version }}
-        - cython
-        - double-conversion
-        - flex
-        - fmt
-        - glog
-        - llvmdev {{ llvm_version }}
-        - llvm {{ llvm_version }}
-        - libarchive
-        - libgdal
-        - libpng
-        - librdkafka
-        - numpy
-        - pyarrow ={{ arrow_version }}=*{{ build_ext }}
-        - python
-        - pytest
-        - tbb
-        - tbb-devel
-        - thrift-cpp {{ thrift_version }}
-        - {{ pin_subpackage('omniscidbe', exact=True) }}
-      run:
-        - arrow-cpp-proc {{ arrow_proc_version }} {{ build_ext }}
-        - pyarrow ={{ arrow_version }}=*{{ build_ext }}
-        - python
-        - tbb4py
-        - {{ pin_subpackage('omniscidbe', exact=True) }}
-    test:
+        - pip check
       requires:
-        - pytest
-        - numpy
-        - pandas
-      imports:
-        - omniscidbe
-      source_files:
-        - Embedded/test/test_exceptions.py
-      commands:
-        - pytest -sv Embedded/test/test_exceptions.py
-
-    about:
-      home: https://www.omnisci.com/
-      license: Apache-2.0
-      license_family: APACHE
-      license_file: LICENSE.md
-      summary: The OmniSci database
+        - pip
 
-      description: |
-        OmniSciDB is an in-memory, column store, SQL relational database
-        that was designed from the ground up to run on GPUs.
-      doc_url: https://www.omnisci.com/docs/latest/
-      dev_url: https://github.com/omnisci/omniscidb
+#  - name: heavydbe
+#    version: {{ version }}
+#    script: build-heavydb.sh
+#    build:
+#      string: {{ build_string_ext }}
+#      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
```

Instead of commenting out a large block of configurations, could you try:
```suggestion
     skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
     skip: true
```